Mathlib/RepresentationTheory/Homological/GroupHomology/LowDegree.lean

```diff
@@ -714,6 +714,10 @@ lemma coinvariantsMk_comp_opcyclesIso₀_inv :
       (chainsIso₀ A).inv ≫ (inhomogeneousChains A).pOpcycles 0 :=
   (CommSq.vert_inv ⟨pOpcycles_comp_opcyclesIso_hom A⟩).w
 
+lemma cyclesMk₀_eq (x : A) :
```

Comment:
I found this lemma confusing to understand because the docstring of `groupHomology.cyclesMk` says that it makes (n+1)-cycle from something in the kernel of something and you're using it to make a 0-cycle. In particular it seems to me that `cyclesMk 0` can be used to make both 0-cycles and 1-cycles! Can you maybe change the docstring of `groupHomology.cyclesMk` to explain this funny use case? I think my confusion stems from the fact that m is implicit, n is explicit, n can be computed from m but m is not uniquely determined by n and hence in my mind the wrong variable is explicit? Or do you feel like you got it right? If you don't want to change explicitness of m and n then maybe just stick to explaining this use case in the docstring?

Reply:
Thank you! I've made the docstring clearer and I've made both `m, n` explicit because it seems to work best that way.